internal/handlers/pg/pgdb/transactions.go

```diff
@@ -26,18 +26,18 @@ import (
 )
 
 // maxRetries is the maximum number of times to retry a transaction.
-const maxRetries = 3
+const maxRetries = 20
 
 // delay is the amount of time to wait before retrying a transaction.
-const delay = 10 * time.Millisecond
+const delay = 3 * time.Millisecond
```

Comment:
Any particular reason we like 3ms?

Reply:
I was trying to come up with a small number (so we don't have big delays), and from the test logs, it looks to me that we don't need 10ms, we can choose a smaller number. Maybe even smaller than 3ms though...

Reply:
For now I think 3ms is fine. Probably we will make change on this once we get more understanding by https://github.com/FerretDB/FerretDB/issues/1686 :+1: